docs/guide/url_management.md

```diff
@@ -1,36 +1,39 @@
 # URL management
 
-## External URL validation
+Manage external URL addresses and URL wildcards with a comfortable user interface that is available in the Back Office, Admin tab, the URL Manager node. Configure URL aliases to have human-readable URL addresses throughout your system.
```

Comment:
Fair enough, but you will need to duplicate/repeat the same information in the user guide as well. I'd suggest a bit more adaptation:
```suggestion
You can manage external URL addresses and URL wildcards in the Back Office, **Admin** tab, the **URL Manager** node. Configure URL aliases to have human-readable URL addresses throughout your system.
```